src/Agoda.Analyzers.Test/AgodaCustom/AG0039UnitTests.cs

```diff
@@ -8,107 +8,51 @@ namespace Agoda.Analyzers.Test.AgodaCustom;
 
 class AG0039UnitTests : DiagnosticVerifier
 {
-    protected override DiagnosticAnalyzer DiagnosticAnalyzer => new AG0039UndocumentedMemberAnalyzer();
-
-    protected override string DiagnosticId => AG0039UndocumentedMemberAnalyzer.DIAGNOSTIC_ID;
+    protected override DiagnosticAnalyzer DiagnosticAnalyzer => new AG0039MethodLineLengthAnalyzer();
 
+    protected override string DiagnosticId => AG0039MethodLineLengthAnalyzer.DIAGNOSTIC_ID;
+    
     [Test]
-    [TestCase(@"
-                /// <summary>
-                /// 
-                /// </summary>
-                public class NotDoc
-                {
-                    /// <summary>
-                    /// 
-                    /// </summary>
-                    public string str1 { get; }
-                    ///// <summary>
-                    ///// 
-                    ///// </summary>
-                    //public const int int2 = 1;
-                    ///// <summary>
-                    ///// 
-                    ///// </summary>
-                    //public void DoesNothing() { }
-                    ///// <summary>
-                    ///// 
-                    ///// </summary>
-                    //public event SampleEventHandler SampleEvent;
-                    ///// <summary>
-                    ///// 
-                    ///// </summary>
-                    ///// <param name=""sender""></param>
-                    //public delegate void SampleEventHandler(object sender);
-                }
-")]
-    [TestCase(@"
-                /// <include file='xml_include_tag.xml' path='MyDocs/MyMembers[@name=""test""]/*' />
-                public class NotDoc
+    public async Task AG0039_At120LinesOfCode_ShowWarning()
```

Comment:
Test name says "at 120", test implementation runs creates 121 lines, and code is configured to warn above 40 :) 

Can we test edge-cases: doesn't warn at 40, but warns at 41?

Reply:
fixed